### DISCUSSION TITLE
[BUG] Form helper hijacked the input checkbox actions.

### DISCUSSION THREAD

**Discussion by ghost** [tabler/tabler#1046] [General]
When you use form helper with checkbox, its hijacked the checkbox action. For example. If you click on the ?, the checkbox is actived.

	<div class="row mb-3">
		<label for="product_review" class="col-sm-2 form-label">"allow customer reviews"
		<span class="form-help ms-2" data-bs-toggle="popover" data-bs-placement="top" data-bs-html="true" data-bs-content="<p>ZIP Code must be US or CDN format.</p>">?</span></label>
		<div class="col-sm-10">
			<input type="checkbox" name="product_review" id="product_review" class="form-check-input">
		</div>
	</div>

**Comment by codecalm** [comment]
it's not a but, it's normal behaviour. You use `label` tag, so everything clicked inside `label` tag will activate checkbox. 

You should use code like that to expected behaviour:

```html
<div class="row mb-3">
   <div class="col-sm-2 form-label">
      <label for="product_review">"allow customer reviews"</label>
      <span class="form-help ms-2" data-bs-toggle="popover" data-bs-placement="top" data-bs-html="true" data-bs-content="<p>ZIP Code must be US or CDN format.</p>">?</span>
   </div>
   <div class="col-sm-10">
      <input type="checkbox" name="product_review" id="product_review" class="form-check-input">
   </div>
</div>
```
